Tests/Unit/MiniAppViewTests.swift

```diff
@@ -0,0 +1,29 @@
+import Quick
+import Nimble
+@testable import MiniApp
+
+class MiniAppViewTests: QuickSpec {
+
+    override func spec() {
+        describe("Mini App view") {
+            context("when initialized with valid parameters") {
+                it("will return MiniAppView object") {
+                    guard let testFileURL = MockFile.createTestFile(fileName: "MiniApp.txt") else {
+                        return
+                    }
+                    let miniAppView = MiniAppView(filePath: testFileURL)
+                    expect(miniAppView).toEventually(beAnInstanceOf(MiniAppView.self))
+                }
+                it("will return same frame sizes") {
+                    guard let testFileURL = MockFile.createTestFile(fileName: "MiniApp.txt") else {
+                        return
+                    }
+                    let viewFrame = CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height)
+                    let miniAppView = MiniAppView(filePath: testFileURL)
+                    miniAppView.frame = viewFrame
+                    expect(miniAppView.frame).toEventually(equal(viewFrame))
```

Comment:
Does it make sense to test that? I'll pass immediately.
Maybe something like `expect(miniAppView.frame.equalTo(viewFrame)).toEventuallyNot(beFalse())`?

Reply:
Totally. But I wanted to use the API's from the Quick and Nimble as much as possible to compare any objects(at least in the testcases)

Reply:
Yeah, unfortunately there is no other way in this case. If you want to keep it the way it is now then ok

Reply:
I don't think "will return same frame sizes" tests anything and can be removed.